sqlglot/parser.py

```diff
@@ -3375,11 +3375,17 @@ def _parse_table(
         bracket = parse_bracket and self._parse_bracket(None)
         bracket = self.expression(exp.Table, this=bracket) if bracket else None
 
+        rows_from = self._match_text_seq("ROWS", "FROM") and self._parse_wrapped_csv(
+            self._parse_table
```

Comment:
Why are we parsing a table here?

Reply:
Because `ROWS FROM` is basically a wrapped CSV collection of set/table functions which are currently parsed with `_parse_table`, so one can reuse it recursively. I believe it also semantically sound as `ROWS FROM` is supposed to gather the rows from many set/table functions in one table. 

Hope that made sense